<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ~~~~~~~~~~~~~~^
        recipe=args.recipe,
        ^^^^^^^^^^^^^^^^^^^
    ...<6 lines>...
        verbose_flag=args.verbose,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
        recipe_name,
    ...<3 lines>...
        fix=fix,
    )
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 831, in lint_file
    self.lint_recipe(
    ~~~~~~~~~~~~~~~~^
        recipe,
        ^^^^^^^
        fix=fix,
        ^^^^^^^^
    )
    ^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 891, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/__init__.py", line 294, in run
    self.check_recipe(recipe)
    ~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.13/site-packages/anaconda_linter/lint/check_syntax.py", line 36, in check_recipe
    has_constraints = constraints.search(spec)
TypeError: expected string or bytes-like object, got 'NoneType'

ERROR conda.cli.main_run:execute(127): `conda run conda-lint /tmp/abs_4el1wvybjx/clone` failed. (See above for error)
</details>

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -2,13 +2,6 @@
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./build-aux
 
 ./configure --prefix="${PREFIX}"
-# Remove test broken in sed 4.4
-sed -i.bak -e 's|testsuite/panic-tests.sh||' Makefile
 make -j${CPU_COUNT} ${VERBOSE_AT}
-# on arm64 some tests are failing due true ... so for now ignore those issue ...
-if [[ "${target_platform}" == osx-arm64 ]]; then
-  make check || true
-else
-  make check
-fi
+make check
 make install

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -2,6 +2,8 @@
 cp $BUILD_PREFIX/share/libtool/build-aux/config.* ./build-aux
 
 ./configure --prefix="${PREFIX}"
+# Remove test broken on CI in sed 4.9
+sed -i.bak -e 's|testsuite/panic-tests.sh||' Makefile
 make -j${CPU_COUNT} ${VERBOSE_AT}
 make check
 make install

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -6,7 +6,6 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: http://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.xz
   sha256: 6e226b732e1cd739464ad6862bd1a1aba42d7982922da7a53519631d24975181
 

